CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.19...3.28 FATAL_ERROR)
```

Comment:
This also should be adjusted in the contrib/tenzir-plugins submodule.

Reply:
Done.